package.json

```diff
@@ -101,15 +101,15 @@
         "webpack-dev-server": "^3.11.2"
     },
     "engines": {
-        "node": ">=10.x",
+        "node": ">=18.x",
         "yarn": ">=1.19.x"
     },
     "scripts": {
         "build": "yarn setup && yarn clean && yarn build:i18n && yarn build:dev",
         "build:ci": "yarn setup && yarn webpack --progress --colors --config build/webpack.config.js",
-        "build:dev": "BABEL_ENV=dev NODE_ENV=dev yarn webpack --progress --colors --config build/webpack.config.js",
+        "build:dev": "BABEL_ENV=dev NODE_ENV=dev NODE_OPTIONS=--openssl-legacy-provider yarn webpack --progress --colors --config build/webpack.config.js",
```

Comment:
Did we verify that this is needed? If so, can we upgrade Webpack to avoid it?

Reply:
It is needed if we don't upgrade Webpack. I was thinking it would be better to upgrade Webpack separately as there could be more issues that can occur with upgrading to Webpack 5?

Reply:
Gotcha. We should make sure to do the upgrade at some point, since the option will be deprecated eventually.

Reply:
Will create ticket for that